lib/config/options/index.ts

```diff
@@ -1289,7 +1289,7 @@ const options: RenovateOptions[] = [
   },
   {
     name: 'branchPrefixOld',
-    description: 'Old Prefix to check for existing PRs.',
+    description: 'Old prefix to check for existing PRs.',
```

Comment:
Just Prefix, seems a bit weird, it should probably be one of these:

- prefix
- `prefix` when referring to an old config option name?